plugins/module_utils/fusion.py

```diff
@@ -166,4 +170,13 @@ def fusion_argument_spec():
                 )
             ],
         ),
+        private_key_password=dict(no_log=False),
     )
+
+
+def fusion_required_by():
```

Comment:
`required_by` might be used by the module code itself and it would required to refactor if we ever change the params in the module. `if module.params["private_key_password"]: module.fail_on_missing_params(["private_key_file"])` may be a better way to check this cross-modules. Even better might be to do this check at the beginning `get_fusion()`, all modules call this, there are some env-related checks anyways and being in one place it won't be forgotten.

Reply:
done